features/exploredash/src/main/java/org/dash/wallet/features/exploredash/ui/ExploreViewModel.kt

```diff
@@ -220,7 +223,7 @@ class ExploreViewModel @Inject constructor(
                                     }
                                     .flatMapLatest { bounds ->
                                         _appliedFilters.postValue(
-                                            FilterOptions(query, territory, payment, selectedRadiusOption)
+                                            FilterOptions(query, territory, payment, selectedRadiusOption.value!!)
```

Comment:
`radius` getter has a null check for the `selectedRadiusOption`, but here it's just assumed that it's not null. Which is likely, but not guaranteed.

It's better to use the non-null value that you already have access to in the flatMap (just need to declare it):
```
_selectedRadiusOption.flatMapLatest { selectedRadius ->
```

This one is guaranteed to be non-null.